install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -59,17 +59,30 @@ serviceAccount:
   name: kubecost-irsa-sa
 ```
 
-Advanced configuration (for larger deployments):
+#### Aggregator Optimizations
+
+For larger deployments of Kubecost, Aggregator can be tuned.
+
+{% hint style="warning" %}
+Aggregator is a memory and disk-intensive process. Ensure that your cluster has enough resources to support the configuration below.
+
+Because the Aggregator PV is relatively small, the least expensive performance gain will be to move the storage class to a faster SSD this varies by provider, the terms used are gp3/extreme/premium/etc.
+{% endhint %}
+
+{% hint style="warning" %}
+{% endhint %}
+
+The settings below are in addition to the basic configuration above.
```

Comment:
@michaelmdresser

Reply:
The settings below are in addition to the basic configuration above

Reply:
Ah! Thank you